server/statistics/hot_peer.go

```diff
@@ -41,6 +41,17 @@ type HotPeerStat struct {
 	isNew      bool
 }
 
+// ID returns region ID. Implementing TopNItem.
+func (stat *HotPeerStat) ID() uint64 {
+	return stat.RegionID
+}
+
+// Less compares two HotPeerStat.Implementing TopNItem.
+func (stat *HotPeerStat) Less(than TopNItem) bool {
```

Comment:
`than` look like strange

Reply:
Learned it from google btree https://godoc.org/github.com/google/btree#Item.